modules/ixBidAdapter.md

```diff
@@ -42,16 +42,21 @@ var adUnits = [{
 ```javascript
 var adUnits = [{
     // ...
-
     mediaTypes: {
         banner: {
             sizes: [
                 [300, 250],
                 [300, 600]
             ]
+        },
```

Comment:
FYI for `3.0` we are removing the `Old Format` support.

So the top level `adUnit.sizes` will not be supported any longer.

So maybe when we move to `3.0` we can update the ix.md file!

Reply:
We are planning a release specifically for 3.0 with this change. 👍